docs/core/deploying/deploy-with-cli.md

```diff
@@ -1,201 +1,154 @@
-
-1. Run your app
+\* When using this SDK version, this is the default Publish Mode when running the basic `dotnet publish` command.
 
-   Once installed, users can execute your application by using the `dotnet` command and providing the application filename, such as `dotnet fdd.dll`.
+## Publishing basics
 
-   In addition to the application binaries, your installer should also either bundle the shared framework installer or check for it as a prerequisite as part of the application installation.  Installation of the shared framework requires Administrator/root access.
+When you publish your app, the default target the app is compiled for (automatically chosen when you create the app) is specified in the `<TargetFramework>` setting of the project file. Choose any valid [Target Framework Moniker (TFM)](~/dotnet/standard/frameworks.md). For example, if you have `<TargetFramework>netcoreapp2.2</TargetFramework>` chosen, an executable that targets .NET Core 2.2 is created. The TFM specified in this setting is the default target used by the [`dotnet publish`][dotnet-publish] command.
 
-## Framework-dependent deployment with third-party dependencies
+You can use the `<TargetFrameworks>` setting (note the plural) to specify more than one framework supported by your app. Separate each TFM with a semicolon. You can publish, targeting one of the frameworks listed, with the `dotnet publish -f <TFM>` command. For example, if you have `<TargetFrameworks>netcoreapp2.1;netcoreapp2.2</TargetFrameworks>`, and run `dotnet publish -f netcoreapp2.1`, an executable that targets .NET Core 2.1 is created. 
 
-Deploying a framework-dependent deployment with one or more third-party dependencies requires that those dependencies be available to your project. Two additional steps are required before you can run the `dotnet restore` ([see note](#dotnet-restore-note)) command:
+Unless otherwise set, the output directory of the [`dotnet publish`][dotnet-publish] command is `<CURRENT-FOLDER>/bin/<BUILD-CONFIGURATION>/<TFM>/publish/`. The default **BUILD-CONFIGURATION** mode is **Debug**. Based on the SDK you're using, and the target framework chosen, either a *framework-dependent deployment* or *framework-dependent executable* app is created.
 
-1. Add references to required third-party libraries to the `<ItemGroup>` section of your *csproj* file. The following `<ItemGroup>` section contains a dependency on [Json.NET](https://www.newtonsoft.com/json) as a third-party library:
+### Native dependencies
 
-      ```xml
-      <ItemGroup>
-        <PackageReference Include="Newtonsoft.Json" Version="10.0.2" />
-      </ItemGroup>
-      ```
+If your app has some sort of native dependency, it may not run on a different operating system. For example, if you (on Windows) referenced a library that used the native Win32 API, your app wouldn't run on macOS or Linux. You would need to provide platform-specific code, and compile an executable for each platform. 
```

Comment:
nit: remove "some sort of"

Reply:
referenced --> reference
used --> uses
wouldn't --> won't 
would need --> need
no comma before "and compile"